crates/proof-of-sql-parser/src/sql.lalrpop

```diff
@@ -373,6 +377,20 @@ pub BooleanLiteral: bool = {
     "false" => false,
 };
 
+TimestampLiteral: IntermediateTimestamp = {
+    "timestamp" <content: STRING_LITERAL> =>? {
+        IntermediateTimestamp::try_from(content.trim_matches('\'').trim())
+            .map_err(|_| User { error: "unable to parse timestamp from query" })
+    },
+};
+
+UnixTimestampLiteral: IntermediateTimestamp = {
+    // Handling the to_timestamp function with numeric input
+    "to_timestamp" "(" <epoch: Int64NumericLiteral> ")" =>? {
+        IntermediateTimestamp::to_timestamp(epoch).map_err(|_| User { error: "unable to parse timestamp from query" })
```

Comment:
We can do that for now but it is pretty odd to have `to_timestamp` as a keyword rather than a function